server/pbench/bin/index-pbench

```diff
@@ -1497,10 +1497,10 @@ def hostnames_if_ip_from_sosreport(sos):
     else:
         pass
 
-    d = {
-        'hostname-f': hostname_f,
-        'hostname-s': hostname_s
-    }
+    d = _dict_const([
+        ('hostname-f', hostname_f),
+        ('hostname-s', hostname_s)
+    ])
```

Comment:
See previous comment.

Reply:
I don't think you can use a "-" in an identifier, right?